src/Core/Suppressions/IClientSuppressionSynchronizer.cs

```diff
@@ -0,0 +1,30 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+namespace SonarLint.VisualStudio.Core.Suppressions
+{
+    public interface IClientSuppressionSynchronizer
```

Comment:
I'm hoping that we'll be able to move the suppressions-related stuff out of `Core` later and into `ConnectedMode` (I'll add a note to the board).
However, it makes sense to put it with the other suppression-related interfaces now.

Reply:
Confused about this statement, Didnt we want this in core?

Reply:
Ideally, all feature-specific stuff would be in feature-specific assemblies, unless that gives us problems with having circular dependencies between them, in which case we'd need to put the interfaces in a "neutral" assembly like Core.
However, it's not a problem for now.